### PR TITLE
Update offboard-machine-api.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details on how to get the ID to offboard

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7477

File: microsoft-365/security/defender-endpoint/offboard-machine-api.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Delegated (work or school account)|Machine.Offboard|'Offboard machine'
 POST https://api.securitycenter.microsoft.com/api/machines/{id}/offboard
 ```
 
+The machine id can be found on the URL when you select the device. Generally, it is a 40 digit alphanumeric number that can be found in the URL.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/offboard-machine-api.md]:
```suggestion
The machine ID can be found in the URL when you select the device. Generally, it is a 40 digit alphanumeric number that can be found in the URL.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**